locale/en_US/locale.po

```diff
@@ -0,0 +1,93 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"POT-Creation-Date: 2022-08-03T15:10:21-07:00\n"
+"PO-Revision-Date: 2022-08-03T15:10:21-07:00\n"
+"Language: \n"
```

Comment:
I have an ongoing PR to update/lint the locale files, if you want to follow my updates, then this should be added:

```po
"Language: en_US\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
```